src/table/selections-utils.js

```diff
@@ -80,3 +80,13 @@ export function selectCell(cell, selState, selDispatch, evt) {
     api.cancel();
   }
 }
+
+export const confirmSelections = (selState) => {
+  const { api } = selState;
+  api.confirm();
+};
+
+export const cancelSelections = (selState) => {
+  const { api } = selState;
+  api.cancel();
```

Comment:
If this is the only thing these functions are suppose to do, they are not needed. just call `selState.api.confirm` from `handleKeyPress`

Reply:
sure